tap_zendesk/http.py

```diff
@@ -1,14 +1,16 @@
 from time import sleep
+from asyncio import sleep as async_sleep
 import backoff
 import requests
 import singer
 from requests.exceptions import Timeout, HTTPError, ChunkedEncodingError, ConnectionError
+from aiohttp import ContentTypeError
 from urllib3.exceptions import ProtocolError
 
 
-
 LOGGER = singer.get_logger()
-
+DEFAULT_WAIT = 60 # Default wait time for backoff
+DEFAULT_WAIT_FOR_CONFLICT_ERROR = 10 # Default wait time for backoff for conflict error
```

Comment:
```suggestion
# Default wait time for backoff
DEFAULT_WAIT = 60

# Default wait time for backoff for conflict error
DEFAULT_WAIT_FOR_CONFLICT_ERROR = 10```

Reply:
More readable and maintains a clear separation between the code and its explanation